recipe/meta.yaml

```diff
@@ -28,6 +27,9 @@ requirements:
     - numpy
     - scipy
     - libpysal
+    - tqdm
+    # FIXME: workaround until we rebuild gdal
+    - zstd
```

Comment:
We should remove this once I fix `gdal`.

Reply:
Ok, thanks again! Should we merge this PR or is it better to way for the `gdal` fix?

Reply:
It is OK to merge. I'm still investigating and a possible rebuild may take days. The odd thing is, a local install of the same version `gdal` does pull `zstd` as expected. I'm not sure what is going on. (If not a lot of things would be broken and not only `segregation`.)